tests/integration/component_test/python-component/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pulumi>=3.150.0,<3.158.0
```

Comment:
Why < 3.158.0?

Reply:
That's not needed anymore.  That was to guard against https://github.com/pulumi/pulumi/pull/18985 breaking the test, but we're installing the SDK from the repo now instead.